docs/scratch_orgs.rst

```diff
@@ -1,38 +1,172 @@
-Manage scratch orgs
+Manage Scratch Orgs
 ===================
 
-What is an org?
----------------
+Throughout this section, we'll focus on scratch org management in the CumulusCI keychain. To learn about managing persistent orgs, such as sandboxes, production orgs, and packaging orgs, read TODO: reference "Connecting persistent orgs".
+
+What is an Org in CumulusCI?
+----------------------------
+
+CumulusCI takes a different approach to creating and using scratch orgs that aims to make the process easier, more portable, and more repeatable. An org in CumulusCI's keychain starts out as a named configuration, tailored for a specific purpose within the lifecycle of the project. The scratch org is only actually generated the first time you use the scratch org from the project keychain - and once it's expired or been deleted, a new one can easily be created with the same configuration.
```

Comment:
> CumulusCI takes a different approach to creating and using scratch orgs that aims to make the process easier, more portable, and more repeatable.

Compared to what? It feels like you're assuming that the reader is familiar with using scratch orgs from sfdx but it's just as likely that the org development model in sandboxes is what they're familiar with.